sourcegraph/cody
# feat: auto-dismiss chat error banner

QOL improvement: dismisses error banner on a set interval

Previously the error banner would stay on the chat view until the user manually clicks the `x`, this change adds an interval to each banner so it dismisses automatically after 3 seconds

## Test plan
- run more than one command sequentially, wait for error banner to pop up in the chat view
- wait 3 seconds, error banner should dismiss automatically (and sequentially if theres more than one!)

Before:

https://github.com/sourcegraph/cody/assets/51868853/f4b233d0-5303-4e0f-b73e-decd269589cf

After:


https://github.com/sourcegraph/cody/assets/51868853/9ef8a154-3be0-4e82-8924-2fb493111030
